src/test/java/guitests/GuiTestBase.java

```diff
@@ -82,17 +87,109 @@ public FxRobot type(KeyCode... keyCodes) {
         return super.type(TestUtil.scrub(keyCodes));
     }
 
-    protected void delay(int milliseconds) {
-        try {
-            Thread.sleep(milliseconds);
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
-    }
-
     protected void pushKeySequence(KeySequence keySequence) {
         push((KeyCodeCombination)keySequence.getKeyCombination());
         push((KeyCodeCombination)keySequence.getSecondKeyCombination());
     }
 
+    protected EditWindow use_PERSON_EDIT_ACCELERATOR() {
```

Comment:
I decided to use the same string here because it is also used as the description field of the KeyBinding object. So this string is used in multiple places, and I didn't want to create multiple versions of it. I'll keep them as they are for now, but keep in mind as a potential refactoring.